R/utils.R

```diff
@@ -68,20 +68,20 @@ path_dropbox_2dii <- function(...) {
 #'
 #' @param path Character vector indicating the directory of a file.
 #'
-#' @return NULL
+#' @return String holding provided `path`.
```

Comment:
See how Jenny does it: https://readr.tidyverse.org/reference/write_delim.html#value
In general, it's useful to clarify that the return value is invisible to the user isn't surprised by the lack of output. Here my comment is FYI because I assume this function is only internal.